raisinbread/migration

```diff
@@ -0,0 +1,32 @@
+#20.2
```

Comment:
what's this file?

Reply:
> added migration record to track(manually) which patches have been run on the dataset

Reply:
because there are PRs open in parallel that are not based on a version that already has raisinbread, some things ar ebeing merged with SQL patches without being run on RB which means at any time, I dont know what has been exectuted and what hasnt